### PR TITLE
Removed preview from documentation set

### PR DESCRIPTION
**Pull request by pagoe-msft**
This is the PR we'll use for .NET and Fixed Version GA on Friday 11/20

### CODE REVIEW

File: microsoft-edge/webview2/concepts/distribution.md [hunk 1]
```diff
@@ -104,15 +104,13 @@ The Web is constantly evolving. The Evergreen WebView2 Runtime is kept up-to-dat
...
-> [!NOTE]
-> The Fixed Version distribution mode is in Public Preview.  
-
-The Fixed Version distribution mode was previously named bring-your-own.  
+## Fixed Version distribution mode   
 
 For constrained environments with strict compatibility requirements, consider using the Fixed Version distribution mode.  Select and package a specific version of the WebView2 Runtime using the Fixed Version distribution mode.  You may specify the timing of Runtime updates for your app.  The Fixed Version distribution mode doesn't receive any automatic updates. You should plan to update your app and the Runtime.  
 
+> [!NOTE] 
+> The Fixed Version distribution mode was previously named bring-your-own.
```
Comment (liminzhu) [inline microsoft-edge/webview2/concepts/distribution.md:112]: nitpick - don't think we need to make this a NOTE, could just leave the previously-named as byo part as is.

**Review by liminzhu — CHANGES_REQUESTED** [review]
Don't forget the roadmap doc :) - https://docs.microsoft.com/en-us/microsoft-edge/webview2/roadmap.

**Review by liminzhu — APPROVED** [review]

File: microsoft-edge/webview2/roadmap.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ The Win32 C/C++ SDK has reached GA.
 
 ### .NET  
 
-The .NET SDK wraps the Win32 C/C++ SDK.  The .NET SDK is expected to GA shortly after the Win32 C/C++ GA in Q4 2020.  
+The .NET SDK wraps has reached GA.
```
Comment (Reezaali) [inline microsoft-edge/webview2/roadmap.md:69]:
```suggestion
The .NET SDK has reached GA. 
```

File: microsoft-edge/webview2/gettingstarted/wpf.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ Start with a basic desktop project containing a single main window.
     
 ## Step 2 - Install WebView2 SDK  
 
-Next add the WebView2 SDK to the project.  For the preview, install the WebView2 SDK using Nuget.  
+Next add the WebView2 SDK to the project using Nuget.
```
Comment (Reezaali) [inline microsoft-edge/webview2/gettingstarted/wpf.md:63]:
```suggestion
Next add the WebView2 SDK to the project using NuGet.  
```

File: microsoft-edge/webview2/gettingstarted/winforms.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ Start with a basic desktop project containing a single main window.
 
 ## Step 2 - Install WebView2 SDK
 
-Next add the WebView2 SDK to the project.  For the preview, install the WebView2 SDK using Nuget.  
+Next add the WebView2 SDK to the project using Nuget.
```
Comment (Reezaali) [inline microsoft-edge/webview2/gettingstarted/winforms.md:45]:
```suggestion
Next add the WebView2 SDK to the project using NuGet.
```

File: microsoft-edge/webview2/concepts/distribution.md (hunk 1 above)
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:112]:
```suggestion
> The Fixed Version distribution mode was previously called bring-your-own.  
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

**Review by Reezaali — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**